src/lib/types/rules.ts

```diff
@@ -1,8 +1,13 @@
+import { Category } from '../enums/category';
+
+export type MetadataDocs = {
+    category: Category;
```

Comment:
Maybe this field should be optional? I'm thinking about the rules that are outside sonar that might have a different category.
Another option will be to have `Category | string` and manually enforce a type when a rule is in core?

Reply:
How do you enforce a type when a rule is in the core?